terraform/variables.tf

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+variable "model_name" {
+  description = "Name of Juju model to deploy application to"
+  type        = string
+  default     = ""
+}
+
+variable "app_name" {
+  description = "Name of the application in the Juju model"
+  type        = string
+  default     = "grafana"
+}
+
+variable "channel" {
+  description = "The channel to use when deploying a charm "
+  type        = string
+  default     = "latest/stable"
+}
+
+variable "grafana_config" {
```

Comment:
Please see the comment above - I'd suggest `grafana_agent_config` or simply `config`.